RELEASE-NOTES.md

```diff
@@ -4,6 +4,38 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.02 release notes (2020-02-18)
+Packages released:
+
+### Components
+- **Data list:** Enabled hidden/visible feature to update display ([#2691](https://github.com/patternfly/patternfly-next/pull/2691))
+- **Data list, compact example:** Added compact version of data list ([#2686](https://github.com/patternfly/patternfly-next/pull/2686))
+- **Data toolbar:** 
+  - Updated input type to search ([#2648](https://github.com/patternfly/patternfly-next/pull/2648))
+  - Made Data toolbar experimental ([#2611](https://github.com/patternfly/patternfly-next/pull/2611))
+- **File upload:** Added File upload component ([#2681](https://github.com/patternfly/patternfly-next/pull/2681))
```

Comment:
**Inline edit:** Fixes Firefox alignment bug
https://github.com/patternfly/patternfly-next/commit/6f56e1655bd31b2c77ebe04d0747ac9b114b5cd3

Reply:
Yes, I'll add it